src/closure-externs/node-externs.js

```diff
@@ -144,3 +144,15 @@ path.isAbsolute;
 path.posix;
 
 crypto.randomFillSync;
+
+
+// d8
+
+var os;
+
+/**
+ * @param {string} cmd
+ * @param {Array.<string>=} args
+ * @return {string}
+ */
```

Comment:
Oops these should go in `src/closure-externs/v8-externs.js`

Reply:
I suppose that does make more sense.